tinman/txgen.py

```diff
@@ -158,13 +156,30 @@ def port_snapshot(conf, keydb, silent=True):
             if num_accounts % 100000 == 0:
                 print("Accounts read:", num_accounts)
     
+    snapshot_file.close()
+    
+    return {
+      "account_names": account_names,
+      "total_vests": total_vests,
+      "total_steem": total_steem,
+      "num_accounts": num_accounts
+    }
+
+def port_snapshot(stats, conf, keydb, silent=True):
```

Comment:
Next update, we need to think about this function. ~130 lines is fine if this is really the best place for it, but feels long.

Reply:
See: https://github.com/steemit/tinman/issues/99